app/assets/javascripts/utils/articles_or_items_utils.js

```diff
@@ -0,0 +1,5 @@
+const getArticlesOrItems = (project) => {
```

Comment:
Consider using the pattern from `CourseUtils.i18n`: you could have a function to which you pass in the message key along with the project. This would also let you more easily solve the problem of a fallback when the new `items` version isn't translated or perhaps doesn't exist; you can use the `default` option to fall back to the `articles` version when needed. (For users, that would mean seeing the "articles" version of a message in their own language rather than seeing the "items" version in English, if there's no translation yet available.)